composer.json

```diff
@@ -67,10 +67,10 @@
     },
     "require-dev": {
         "bamarni/composer-bin-plugin": "^1.3",
-        "infection/infection": "^0.10",
+        "infection/infection": "^0.15",
         "mikey179/vfsstream": "^1.6",
-        "phpunit/phpunit": "^8.1",
-        "symfony/phpunit-bridge": "^4.2 || ^5.0"
```

Comment:
out of curiosity why removing the bridge? I think its deprecation helper is quite helpful

Reply:
I thought the library was not used at all. I didn't think about the deprecation helper.
I can make another PR to put it back if you wish.

Reply:
That would be nice 👍

Reply:
Done #475